lib/start.js

```diff
@@ -94,7 +94,8 @@ const start = (passthroughArgs, buildConfig = config.defaultBuildConfig, options
     stdio: 'inherit',
     timeout: options.network_log ? 120000 : undefined,
     continueOnFail: options.network_log ? true : false,
-    shell: process.platform === 'darwin' ? true : false
+    shell: process.platform === 'darwin' ? true : false,
+    killsignal: options.network_log && process.env.RELEASE_TYPE ? 'SIGKILL' : 'SIGTERM'
```

Comment:
this should be `killSignal`: https://nodejs.org/api/child_process.html#child_process_child_process_spawnsync_command_args_options

Reply:
my bad, thanks for the catch